### DISCUSSION TITLE
Client ID and Login

### DISCUSSION THREAD

**Discussion by IsMPS** [spotify-web-api-java/spotify-web-api-java#342] [Q&A]
Hi i read the Documentation but i dont really know how to use more than my user, im searching in the code and i find that the Client ID maybe can be the user and in the javascript you can use the login from spotify but here in java i dont know how to use it. Maybe im stupid but i want for my school, im learning and i cant find where i can change the user.

**Comment by dargmuesli** [comment]
The client authentication flow authenticates a client, not a user. See https://developer.spotify.com/documentation/web-api/concepts/authorization#which-oauth-flow-should-i-use to find that you might rather want to use the authorization code flow.
